src/omero_zarr/raw_pixels.py

```diff
@@ -211,6 +211,12 @@ def add_group_metadata(
     zarr_root.attrs["multiscales"] = multiscales
     zarr_root.attrs["omero"] = image_data
 
+    try:
+        if image._re is not None:
```

Comment:
Any reason not to use `image._closeRE()` which seem to wrap this entire logic?

Reply:
Good idea. Pushed a fix.